modules/programs/yazi.nix

```diff
@@ -39,6 +39,21 @@ let
       rm -fp $tmp
     }
   '';
+
+  xonshIntegration = ''
+    def __yazi_init():
+      def ya(args):
```

Comment:
It's better to use `yy` instead of `ya` here, since `ya` has been used as the CLI program name in the new version of Yazi. See https://github.com/sxyazi/yazi/issues/914 for details.

Reply:
Fixed for xonsh integration. Though I wonder whether it should be done in this PR, as it does not cover other yazi integrations.

Reply:
It should be done in this PR - the new integrations should avoid using `ya`, and the upcoming Yazi v0.3 has marked this as a breaking change https://github.com/sxyazi/yazi/issues/1046